dev-client/src/screens/ProjectTeamScreen/components/RolePill.tsx

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright © 2024 Technology Matters
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see https://www.gnu.org/licenses/.
+ */
+
+import {useTranslation} from 'react-i18next';
+
+import {ProjectMembership} from 'terraso-client-shared/project/projectSlice';
+
+import {Badge} from 'terraso-mobile-client/components/NativeBaseAdapters';
+
+type RolePillProps = {
+  membership: ProjectMembership;
+};
+
+export const RolePill = ({membership}: RolePillProps) => {
+  const {t} = useTranslation();
+  return (
+    <Badge
```

Comment:
I've also asked @CourtneyLee333 if it make sense to call this Chip in the Component spec, since that's what Material UI calls it. (Apple doesn't have an equivalent item; Token is closest.)

Reply:
We have a separate ticket to standardize chips / pills / whatever in https://github.com/techmatters/terraso-mobile-client/issues/1877